syntactic-whitelist/build.sbt

```diff
@@ -0,0 +1,9 @@
+name := "syntactic-whitelist"
+
+version := "0.1"
+
+scalaVersion := "2.13.8"
+
+libraryDependencies += "org.scalameta" %% "scalameta" % "4.5.0"
+libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % Test
+libraryDependencies += "junit" % "junit" % "4.13.2"
```

Comment:
It'll be better if you can merge the `build.sbt` file into the one under the root directory to keep all definitions in a single file.

Reply:
Solved during the meeting